content/security/advisories.html.haml

```diff
@@ -31,3 +31,17 @@ section: security
                   = page.title
                   - if page.kind
                     = "(#{page.kind})"
+                - if page.issues && year == '2018'
+                  %br
+                  %small
+                    %ul{:style => 'padding-left: 10px;'}
+                      - if page.core
+                        %li
+                          Jenkins (core)
+                      - plugins = page.issues.collect { |issue| issue.plugins }.flatten.uniq.keep_if { |p| p }.sort { |x,y| site._generated[:update_center].plugins[x.name].title <=> site._generated[:update_center].plugins[y.name].title }
+                      - if plugins.size > 0
+                        %li
+                          Included Plugins:
+                          - plugins.each_with_index do | plugin, idx |
+                            %a{:href => 'https://plugins.jenkins.io/' + plugin.name, :style => 'white-space: nowrap; padding: 2px;'}
+                              = site._generated[:update_center].plugins[plugin.name].title
```

Comment:
Possible to add a comma between elements of the list ? even they have some paddings around them, I think it could be easier to separate mentally the different plugins.

```
- if index != plugins.size - 1
  = ", "
```